.github/workflows/on-demand-vrfv2plus-smoke-tests.yml

```diff
@@ -70,6 +70,8 @@ jobs:
     with:
       custom_test_list_json: ${{ needs.set-tests-to-run.outputs.test_list }}
       chainlink_version: ${{ inputs.chainlink_version }}
+      slack_notification_after_tests: always
+      slack_notification_after_tests_name: VRF V2 Plus Smoke Tests
```

Comment:
can you add the name of the config here as well? So that its more clear what run it is straight away from Slack message